README.md

```diff
@@ -215,6 +215,7 @@ Currently **officially** using Amundsen:
 1. [Remitly](https://www.remitly.com/)
 1. [Square](https://squareup.com/us/en)
 1. [Workday](https://www.workday.com/en-us/homepage.html)
+1. [WeTransfer](https://wetransfer.com)
```

Comment:
thanks @higee ! the list is sorted alphabetically, could you put it before workday?

Reply:
@feng-tao my bad. Pushed a new commit :)

Reply:
no worry, thanks!